interface/migrations/0019_auto_20200204_2118.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.0.2 on 2020-02-04 19:18
```

Comment:
Can you please rename the migration to something descriptive like `0019_assignment_language`?